tests/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required (VERSION 3.9)
 project(acl_unit_tests)
 
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Let's use 11 for now. Some compilers still barely support it... like android with nvidia codeworks. I think android might be the only platform dragging behind for this that we care about.

Reply:
Make your code C++11 first then:
```
In file included from /home/janisozaur/workspace/acl/tools/acl_compressor/sources/main.cpp:25:
/home/janisozaur/workspace/acl/tools/acl_compressor/../../includes/acl/core/memory.h:321:4: warning: use of this statement in a constexpr function is a C++14 extension [-Wc++14-extensions]
                        ACL_ENSURE(is_valid(), "Invalid PtrOffset!");
                        ^
/home/janisozaur/workspace/acl/tools/acl_compressor/../../includes/acl/core/error.h:72:47: note: expanded from macro 'ACL_ENSURE'
                #define ACL_ENSURE(expression, format, ...) acl::error_impl::assert_impl(expression, format, ## __VA_ARGS__)
                                                            ^
In file included from /home/janisozaur/workspace/acl/tools/acl_compressor/sources/main.cpp:25:
/home/janisozaur/workspace/acl/tools/acl_compressor/../../includes/acl/core/memory.h:328:4: warning: use of this statement in a constexpr function is a C++14 extension [-Wc++14-extensions]
                        ACL_ENSURE(is_valid(), "Invalid PtrOffset!");
                        ^
/home/janisozaur/workspace/acl/tools/acl_compressor/../../includes/acl/core/error.h:72:47: note: expanded from macro 'ACL_ENSURE'
                #define ACL_ENSURE(expression, format, ...) acl::error_impl::assert_impl(expression, format, ## __VA_ARGS__)
                                                            ^
```

Reply:
indeed, if those don't compile as constexpr, we should make them inline for now and add a comment to that effect like I did in too many places now... support for c14 can't come soon enough!

Reply:
note that I only compiled the decompression on android which might be why this didn't pop out yet

Reply:
worth nothing that the nvidia codeworks pipeline that ue4 and stingray and so many others use appears to be deprecated/stale/dead so it's unclear which path they will take or when.